calico-policies/public-network-isolation/eu-de/allow-public-services-pods.yaml

```diff
@@ -32,7 +32,6 @@ spec:
       - 159.122.102.38
       - 161.156.102.206
       ports:
-      - 443
```

Comment:
Removed unnecessary 443 port based on docs here (only 6443 is listed for public in table): https://cloud.ibm.com/docs/Monitoring-with-Sysdig?topic=Sysdig-network#network_outgoing_traffic